### DISCUSSION TITLE
Which system is used to quote ra and dec values?

### DISCUSSION THREAD

**Discussion by cwjames1983** [chime-frb-open-data/community#29] [Q&A] [answered]
## Expected Behavior

The coordinate system used to quote ra and dec should be specified (e.g. J2000) in the "Information" tab.

## Current Behavior

No coordinate system is given - just described as "declination" and "right ascension"

## Possible Solution

Add epoch of measurement to the appropriate row in the "information" tab

## Steps to Reproduce (for bugs)

N/A not a bug

## Context

Creates small offsets in positions if the wrong system is used.

## Your Environment

* Python Version:
* Environment name and version (e.g. Chrome 39,):
* Operating System and version (desktop or mobile):
* Link to your project:

N/A (not a bug)

**Comment by chitrangpatel** [comment, marked as answer]
This has now been fixed.
<img width="598" alt="Screen Shot 2021-07-27 at 3 07 36 PM" src="https://user-images.githubusercontent.com/11685590/127213283-72414291-db9e-43f4-aa97-c08c0b6b63b0.png">

**Comment by shinybrar** [comment]
Re-opening the issue as @cwjames1983 should close it when they are satisfied with the answer.

**Comment by cwjames1983** [comment]
Thanks shinybar - as it turns out, I am satisfied with the answer, but thanks very much for enforcing protocol! And thanks Chitrang for the fix. I guess this was an easy one :-)
